### DISCUSSION TITLE
Tailwind Install Help

### DISCUSSION THREAD

**Discussion by MrSaturn00** [nunocoracao/blowfish#1206] [Q&A]
I am trying to install tailwind to my website that has already had quite a bit of content already added, but when I run the command I get this and it seems to be idle. It hasn't moved in 10 minutes and I have this warning? Did I do something wrong?
![issue](https://github.com/nunocoracao/blowfish/assets/66644553/a2df1d70-3381-4ca2-aa9c-0bf04ab96121)

**Comment by knowinginner** [comment]
Hello. It's useful to search keyword tailwind in [blowfish.page tailwind](https://blowfish.page/docs/advanced-customisation/) github discussion and pull requests.
I have posted while back 
`cd themes/blowfish
npm install`
`cd ../..
./themes/blowfish/node_modules/tailwindcss/lib/cli.js -c ./themes/blowfish/tailwind.config.js -i ./themes/blowfish/assets/css/main.css -o ./assets/css/compiled/main.css --jit`
As to the grid I'll use blowfish layout grid for card view with full width, standard list view, single.html layout. I would like to add that theme is using [500 free tailwind components](https://tw-elements.com/) and some of them will still need a bit of care.
